src/nvim/ex_docmd.c

```diff
@@ -7626,8 +7626,25 @@ static void ex_checkhealth(exarg_T *eap)
     mods_len = add_win_cmd_modifiers(mods, &cmdmod, &multi_mods);
     assert(mods_len < sizeof(mods));
   }
+
+  char *plugin_name  = NULL;
+  char *e = strstr(eap->arg, "++f");
```

Comment:
I think we've discussed a `:float` or `:popup` command that works like `:split` and `:tab`. Don't want a health-specific special case here.

Meanwhile to unblock this pr, for now we could support a `vim.g.health = { style = 'float' }` option.

Reply:
i have also consider a modifier like `float` but unsure how this modifier work bause position can be also configed?